main.cpp

```diff
@@ -0,0 +1,228 @@
+// Configuration editor
+// Command line utiltiy to create configuration files
+// Valid arguments are init and edit
+
+#include <iostream>
+#include <string>
+#include <fstream>
+#include <stdio.h>
+
+
+using std::cout;
+using std::endl;
+using std::string;
+using std::cin;
+
+const string CONFIG_FILE_PATH = "/users/bconway/SourceTree/School/CPT-180-27-Assignment-2-Config-Generator/app.cfg";
+
+
+int main(int argc, char *argv[]) {
+
+    string argument1 = "";
+    // Make sure that the argument isn't empty
+    if (argv[1] != nullptr) {
```

Comment:
@bconway02 That's would be a nice feature! I was planning something similar where if the user would give the wrong argument for the field to edit or had a typo it would list all the possible options again and then allow the user to enter the field before running the edit function. But I seem to be having a hard enough time getting the file to read/ write correctly when editing so I sort of put that feature on the back burner until I can get the main stuff fixed. But you seem to really understand this, I bet you could get your idea to work :)